dom/src/MainDOMSource.ts

```diff
@@ -204,6 +206,7 @@ export class MainDOMSource implements DOMSource {
   }
 
   dispose(): void {
+    this._sanitation$.shamefullySendNext('');
```

Comment:
Do we need to send an empty string through? Looks like you can call `shamefullySendNext` with no arguments. (Unless this is just to make TypeScript happy, ignore me if so).